Gemfile

```diff
@@ -50,8 +50,12 @@ gem 'rinku'
 gem 'rouge', '~> 3'
 gem 'ruby-oembed'
 
+# to help obtaining app wide URI that uniquely identifies model instance
+# (used in notify_identifier for NOTIFY/LISTEN to jobs)
+gem 'globalid'
+
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder'
+gem 'jbuilder', '2.12.0'
```

Comment:
why lock this version?

Reply:
this one is purely oversight. I did this so that the `bundle install` command can fetch the latest version, but I did not revert it back once it's done. Will take note of this in the future not to leave it locked